### DISCUSSION TITLE
Best way to update UniFi Network Server LXC

### DISCUSSION THREAD

**Discussion by up2you1** [tteck/Proxmox#3890] [Help Needed] [answered]
I installed the UniFi Network Server LXC script last month and all working well, thank you! i notice there is no an update, whats best way to update the LXC? re run the script? 

thanks.

**Comment by MickLesk** [comment, marked as answer]
If you hit "update" in LXC console - nothing happens?

Alternative: Copy the Install bash in the LXC console and hit enter

**Reply by up2you1** [reply]
thank you! was a simple as typing in update in the lxc console.
